### ISSUE TITLE
Under Ubuntu 18.x running as normal user, silent permission error, no results return.

### ISSUE THREAD

**Issue by tdukai** [sebhildebrandt/systeminformation#163]
### Issue Type

* [X] Bug
* [ ] Feature Request
* [X] Question

### Expected behavior 
Return system information or at least a permission error

### Actual behavior
Returning empty fields

### Steps to reproduce the problem

```javascript
try {
        const system = await si.system();
        const bios = await si.bios();
        const board = await si.baseboard();
        console.log('SYSTEM:', system);
        console.log('BIOS:', bios);
        console.log('BOARD:', board);
    } catch (err) {
        console.error(err);
    }
```
### Specifications 

- package version: 3.45.7
- operating system: Ubuntu 18.04.1 LTS
- hardware: HP Pavilion dv7
- user: normal user not in SUDO mode

### Problem and possible solution
Debugged the code and it shows a silent permission error. I try to use the component as a basis a license check, where hardware data generated into a hash. I cannot run SUDO mode for the customers.

**Comment by sebhildebrandt** [comment]
@tdukai , can you check, which of the three System Information calls (`si.system()`, `is.bios()`, `is.baseboard()`) throws the silent permission error?

**Comment by sebhildebrandt** [comment]
@tdukai: 

- there are sole permission errors that will throw some console.logs ... I now handle this correctly it (will be released in version 4.0 which will be out end of January) 
- some functions will ONLY run with admin privileges, so I can't do it differently.
- if you need a unique ID, I added a new si.uuid() function which returns os specific uuid (per installation). Maybe this is useful for your purposes. As this returns an object, I plan to extend this also to add some hardwire specific UUIDs.

**Comment by sebhildebrandt** [comment]
Should work now (s described above). Version 4.0.0 published
